p/scripts/main.js

```diff
@@ -859,13 +859,17 @@ function init_stream(stream) {
 
 		el = ev.target.closest('.item.share > a[href="#"]');
 		if (el) {	//Print
-			const content = '<html><head><style>' +
-				'body { font-family: Serif; text-align: justify; }' +
-				'a { color: #000; text-decoration: none; }' +
-				'a:after { content: " [" attr(href) "]"}' +
-				'</style></head><body>' +
-				el.closest('.flux_content').querySelector('.content').innerHTML +
-				'</body></html>';
+			let cssUrl = [];
+			for (var i = 0; i < document.styleSheets.length; i++){
+				cssUrl.push(document.styleSheets[i].href);
+			}
+			const cssLinkReducer = (carry, item) => `${carry}<link href="${item}" rel="stylesheet" type="text/css" />`;
+			const content = `
```

Comment:
Arrow functions and ES6 template literals are not supported by IE11
https://caniuse.com/#feat=arrow-functions
https://caniuse.com/#feat=template-literals

Reply:
Damn it :'(